src/components/manage/Blocks/Listing/Edit.jsx

```diff
@@ -48,7 +48,7 @@ const Edit = React.memo(
         : intl.formatMessage(messages.items));
 
     return (
-      <>
+      <div className={props.className}>
```

Comment:
If we wanted to be super strict, this would be a breaking change, so it would need to be tracked, but it would be ok to release it since this is going into 16 alpha.

Reply:
True, let's leave it unchanged now.